DBImport/README.md

```diff
@@ -0,0 +1,33 @@
+## Repo Summary
```

Comment:
do you think we should put this together under the root README?

Reply:
Let's do it